projects/lendai-us/design-system/src/lib/phone-input/phone-input.component.ts

```diff
@@ -56,6 +61,10 @@ import {
     MatButtonModule,
     IconComponent,
     MatMenuModule,
+    FilterPipe,
```

Comment:
i don't see usage of filter pipe and reactive forms module, if they indeed aren't used, we can remove their imports